package.json

```diff
@@ -20,17 +20,18 @@
         "babel-loader": "^8.2.2",
         "bootstrap": "^4.6.0",
         "cache-loader": "^1.0.3",
-        "html-loader": "^0.5.5",
-        "html-webpack-plugin": "^3.2.0",
+        "html-loader": "^5.0.0",
```

Comment:
Reminds me that we need to keep all of our dependencies up to date... locking versions seems like a mistake?